docs/sources/tempo/configuration/_index.md

```diff
@@ -1204,6 +1202,54 @@ memberlist:
 
 ```
 
+## Configuration Blocks
+
+Defines re-used configuration blocks
+
+### WAL Config
+
+The Storage WAL configuration block.
+
+```yaml
+# Where to store the wal files while they are being apeended to.
+# Must be set.
+# Example: "/var/tempo/wal
+[path: <string> | default = ""]
+
+# Where to store the completed wal files
+# If not set (""), will join the `path` with "completed" to generate the effective path
+# Example: "/var/tempo/wal/completed"
+[completedfilepath: <string> | default = join(.path, "/completed")]
+
+# Where to store the intermediate blocks while they are being apeended to.
+# Will always join the `path` with "blocks" to generate the effective path
+# Example: "/var/tempo/wal/blocks" (ignored)
+[blocksfilepath: <ignored> | = join(.path, "/blocks")]
```

Comment:
I think I found a bug?

I don't think this is the intended behavior

Please verify :
https://github.com/grafana/tempo/blob/251bf5a9a74fd586ffd3dedcc2c8000f36aa9a78/tempodb/wal/wal.go#L75-L81

Reply:
@kvrhdn 

Could you take a look at this?
I described what the code does, but I don't think this is intended.

I could add the `if` block but this is supposed to be docs so...